package auth.base.user.exception;

public class GlobalExceptionHandler {
}
